### ISSUE TITLE
New Page Should use semi-colon namespace without the folder name

### ISSUE THREAD

**Issue by elibroftw** [unoplatform/uno#12159]
## What would you like to be added:

When create a new blank page using the uno template,  it'll produce the following output:

```cs
namespace App.Views {
    public sealed partial class NavigationPage : Page {
        public NavigationPage() {
            this.InitializeComponent();
        }
    }
}
```

This causes bugs when building related to "InitializeComponent" . What I want is 

```cs

namespace App; 

public sealed partial class NavigationPage : Page {
    public NavigationPage() {
        this.InitializeComponent();
    }
}
```

## For which Platform:

- [ ] WebAssembly
- [ ] Android
- [ ] iOS
- [ ] macOS (AppKit)
- [ ] Mac Catalyst
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Linux Framebuffer
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [x] Solution Templates

**Comment by Youssef1313** [comment]
> This causes bugs when building related to "InitializeComponent"

Can you clarify more? 

Can you also try adding `csharp_style_namespace_declarations = file_scoped` to .editorconfig?

**Comment by Youssef1313** [comment]
To me it looks like these errors are unrelated to the use of file-scoped vs block-scoped namespaces.

So is this issue about two different issues?

For the file-scoped part, I guess Roslyn is supposed to respect that when creating new files. I would suggest first reporting to dotnet/roslyn

The implementation in Roslyn was done in https://github.com/dotnet/roslyn/pull/55504.

**Comment by Youssef1313** [comment]
> The .editorconfig change had no impact to how the blankpage looks like.

That looks like a Roslyn bug. So I recommend first opening an issue there. As mentioned, the functionality was supposed to be implemented in https://github.com/dotnet/roslyn/pull/55504 (cc @davidwengier)

**Comment by Youssef1313** [comment]
For the ordering of the item templates, I'm not sure. @dansiegel is this something we control?

**Comment by DevTKSS** [comment]
@MartinZikmund I would not know this happened since I begann using uno. New Page templates are working flawless for me 🤔 maybe outdated as Uno does not longer depend on UWP?

**Comment by MartinZikmund** [comment]
Indeed, thanks to `.editorconfig` Visual Studio correctly adjusts the namespaces
